lib/GallerySwiper.js

```diff
@@ -89,7 +89,7 @@ var GallerySwiper = function (_Component) {
                 newLazyload = nextProps.lazyLoad;
 
 
-            if (images !== newImages) {
+            if (images !== newImages || images.length !== newImages.length) {
```

Comment:
is there a use case for this? Are you changing the original list somehow instead of providing a new list?